packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx

```diff
@@ -235,6 +237,13 @@ const TranslationInput = (props: TranslationInputProps) => {
           </>
         )}
       </Row>
+      {props.additionalInfo && (
+        <Row>
+          <LeftColumn>
+            <InfoMessage>{props.additionalInfo}</InfoMessage>
+          </LeftColumn>
+        </Row>
+      )}
```

Comment:
I think rendering the additional info should be placed where the errors and warning are (around line 199) so no new `Row` component is needed.

Reply:
See my other comment about that